library/src/scala/tasty/Reflection.scala

```diff
@@ -3313,20 +3313,298 @@ trait Reflection { reflection =>
   //   UTILS   //
   ///////////////
 
-  /** TASTy Reflect tree accumulator */
-  trait TreeAccumulator[X] extends reflect.TreeAccumulator[X] {
-    val reflect: reflection.type = reflection
-  }
+  /** TASTy Reflect tree accumulator.
```

Comment:
I remember one of the previous PRs is to avoid the name `TASTy`. 
```suggestion
  /** Tree accumulator.
```

Reply:
I'll do a general cleanup of the docs in a separate PR